database/migrations/2015_11_04_015813_create_link_table.php

```diff
@@ -22,13 +22,13 @@ public function up()
             $table->string('short_url');
             $table->longText('long_url');
             $table->string('ip');
-            $table->string('creator');
+            $table->string('creator')->nullable();
             $table->string('clicks')->default(0);
-            $table->string('secret_key');
+            $table->string('secret_key')->nullable();
```

Comment:
We should create a new migration rather than editing an existing migration. Otherwise, existing instances won't be able to get updates and the change hurts compatibility.

https://laravel.com/docs/5.1/migrations